lesson/day01.md

```diff
@@ -140,134 +146,31 @@ msg:
 
-Put `TASM.EXE, TASM80.TAB, TI83PLUS.INC, DEVPAC8X.COM `here
-
-`Exec`
-
-Look here for compiled programs
-
-In the `Tasm` folder, make a new text file and type in this:
-
-    
-```batch
-@echo off
-echo ==== Now assembling %1.z80 for the TI-83 Plus ====
-tasm -80 -i -b c:\asm\source\%1.z80 c:\asm\exec\%1.bin
-if errorlevel 1 goto ERRORS
-rem This is necessary because of a DevPac8x bug
-cd c:\asm\exec
-c:\asm\tasm\devpac8x %1
-cd c:\asm\tasm
-echo ==== Job finished. Program saved as %1.8xp ====
-goto DONE
-:ERRORS
-echo ==== Errors!!! ====
-:DONE
-del c:\asm\source\%1.lst > NUL
-del c:\asm\exec\%1.bin > NUL
-echo ==== Done ====
-```
-
-And save as `asm.bat`. What you just made is called a batch file and is
-similar in purpose to TI-BASIC programs.
-
-## Sample Z80 Program
-
-Now to make sure that everything has been set up satisfactorialy, we are going
-to write, assemble, link, and send a little do-nothing program. Enter the
-following source code and don't bother trying to understand it... yet. And
-save as `hello.z80` in the `source` directory  
-to compile, open up DOS (try Start menu, Run, then whichever of `command.com`
-or `cmd.exe` works) and go to the TASM directory. Type `asm hello` and press
-Enter.
-
-After a second or two (or more, depending on your computer's speed), assembly
-will finish, and the program is ready to be transmitted to the calculator.
-
-### Navigating Within DOS
-
-Since Windows completely replaced DOS as the operating system for PCs, most
-newer computer users will be at somewhat of a loss when confronted with the
-DOS command prompt. Here are a few things about DOS that you will find helpful
-when navigating through your directory structure. I presume that you have
-enough experience with file managers like Explorer to know what things like
-"subdirectory" and "parent directory" mean.
-
-Command Prompt
-
-At the extreme left of the screen is the command prompt. This is the name of
-the current drive, current directory, and all parent directories. The format of
-internet URLs are directly based on the command prompt (although not
-specifically the DOS prompt), so you shouldn't have any trouble interpreting
-it.
-
-`cd _directory_`     Changes the current directory. The directory changed to
-must be a subdirectory of the current one.
-
-`cd ..`     Moves to the parent directory of the current directory.
-
-`cd \`     Moves to the root directory.
-
-`dir /p`     Displays a list of all files in the current directory. Useful for
-getting your bearings.
-
-## Sending to the Calculator
-
-Start the Graph Link software. Other programs like TI-Connect I am unfamiliar
-with and cannot give any help. Click on `_L_ink`, `_S_end To >`, then `_R_AM`.
-Navigate to the `C:\Asm\Exec` folder and send `hello.8xp` over.
-
-To run the program, paste `Asm(` from the catalog and `HELLO` from the PRGM
-menu...
-
-And you should get this:
+To run the program, transfer the compiled program to an emulator, using either the Graph
```

Comment:
Should be precise with terminology! We're always assembling, not compiling.

Reply:
Quite right, sir! We are now assembling.